build.rs

```diff
@@ -3,6 +3,7 @@ use std::path::PathBuf;
 
 fn main() {
     let mut builder = cc::Build::new();
+    let target = env::var("TARGET").unwrap_or("".to_string());
```

Comment:
I think `env::var("TARGET")?` as in micro-ecc-sys should work here (TARGET is always set).